server/src/services/api/scannerApi.js

```diff
@@ -124,6 +125,18 @@ async function scannerApi(
     const cache = userCache.has(user.id)
       ? userCache.get(user.id)
       : { coordinates: 0, requests: 0 }
+    const scanNextOptions = {
+      routes: config.scanner.scanNext.routes,
+      showcases: config.scanner.scanNext.showcases,
+      pokemon: config.scanner.scanNext.pokemon,
+      gmf: config.scanner.scanNext.gmf,
+    }
+    const scanZoneOptions = {
+      routes: config.scanner.scanZone.routes,
+      showcases: config.scanner.scanZone.showcases,
+      pokemon: config.scanner.scanZone.pokemon,
+      gmf: config.scanner.scanZone.gmf,
+    }
```

Comment:
Please move these lines up above the function declaration (where `backendConfig` is defined), since they're constant and don't need to be redeclared for every request. Also please use the same config syntax as the `backendConfig` definition uses. 

e.g.
```js
    const scanNextOptions = {
      routes: config.getSafe('scanner.scanNext.routes'),
      ...
    }
```